### PR TITLE
Create universite-catholique-de-louvain-histoire.csl

### PR DESCRIPTION
**Pull request by pbieswal**
Documentation site link added

### CODE REVIEW

File: universite-catholique-de-louvain-histoire.csl [hunk 1]
```diff
@@ -0,0 +1,388 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize="false" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="fr-FR">
+  <info>
+    <title>Université Catholique de Louvain - Histoire (French)</title>
+    <title-short>UCL</title-short>
+    <id>http://www.zotero.org/styles/universite-catholique-de-louvain-histoire</id>
+    <link href="http://www.zotero.org/styles/universite-catholique-de-louvain-histoire" rel="self"/>
+    <link href="https://wordpress.com/post/lesvoyagesdebalnibarbi.wordpress.com/16" rel="documentation"/>
```
Comment (rmzelle) [inline universite-catholique-de-louvain-histoire.csl]: @pbieswal, is there no official documentation we can link to? (even if it's e.g. just to a catalogue entry of a printed style guide)

Your link also asks me to log in. Did you mean to link to https://lesvoyagesdebalnibarbi.wordpress.com/2018/02/17/zotero-csl-ucl/ ?

**Comment by pbieswal** [comment]
Hello,


Thank you to point on the bad link, I corrected it and submltted a new Request.


Regards


PB

________________________________
De : Rintze M. Zelle <notifications@github.com>
Envoyé : dimanche 25 février 2018 05:41:59
À : citation-style-language/styles
Cc : Pierre-Edouard Bieswal; Mention
Objet : Re: [citation-style-language/styles] Create universite-catholique-de-louvain-histoire.csl (#3353)


@rmzelle commented on this pull request.

________________________________

In universite-catholique-de-louvain-histoire.csl<https://github.com/citation-style-language/styles/pull/3353#discussion_r170441318>:

> @@ -0,0 +1,388 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize="false" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="fr-FR">
+  <info>
+    <title>Université Catholique de Louvain - Histoire (French)</title>
+    <title-short>UCL</title-short>
+    <id>http://www.zotero.org/styles/universite-catholique-de-louvain-histoire</id>
+    <link href="http://www.zotero.org/styles/universite-catholique-de-louvain-histoire" rel="self"/>
+    <link href="https://wordpress.com/post/lesvoyagesdebalnibarbi.wordpress.com/16" rel="documentation"/>


@pbieswal<https://github.com/pbieswal>, is there no official documentation we can link to? (even if it's e.g. just to a catalogue entry of a printed style guide)

Your link also asks me to log in. Did you mean to link to https://lesvoyagesdebalnibarbi.wordpress.com/2018/02/17/zotero-csl-ucl/ ?

**Comment by adam3smith** [comment]
Thanks! For future reference, please open only one pull request for a style. You can update your original pull request as described above ("To update this pull request..."). With multiple attempts it's hard to tell what you're actually changing between them.

**Comment by pbieswal** [comment]
Sorry for the duplicate and thanks for the help.

________________________________
De : Sebastian Karcher <notifications@github.com>
Envoyé : samedi 17 mars 2018 03:57:50
À : citation-style-language/styles
Cc : Pierre-Edouard Bieswal; Mention
Objet : Re: [citation-style-language/styles] Create universite-catholique-de-louvain-histoire.csl (#3353)


Thanks! For future reference, please open only one pull request for a style. You can update your original pull request as described above ("To update this pull request..."). With multiple attempts it's hard to tell what you're actually changing between them.

**Pull request merged**